test/java/seedu/addressbook/parser/NameIsSimilarTest.java

```diff
@@ -8,7 +9,17 @@
 
 public class NameIsSimilarTest {
 
+    Name name1;
     
+    @Before 
+    public void setup() throws IllegalValueException{
+        name1 = new Name("John");
+    }
+    
+    @Test
+    public void null_returnsFalse(){
+        assertFalse(name1.isSimilar(null));
```

Comment:
You can also add an argument like this `assertFalse("Both are null", name1.isSimilar(null));`